lib/prawn-rails/rails_helper.rb

```diff
@@ -17,5 +13,46 @@ def prawn_document(options={})
       pdf.render
     end
 
+    def set_prawn_headers
+      # Ignore headers when we're not rendering from an ActionController context.
+      return unless defined?(controller) && controller.respond_to?(:response) && !controller.response.nil?
+
+      options = get_prawn_options
+
+      # Don't override the 'Content-Disposition' if we've chosen to set it elsewhere.
+      controller.response.headers['Content-Disposition'] ||= "#{options[:disposition]}; filename=\"#{options[:filename]}\""
```

Comment:
This `||=` was the biggest pain point we had - we set our own `Content-Disposition`s in our controllers, and having the gem totally scrap those was frustrating!